pegasus/sites.v3/code.org/public/hourofcode/overview.haml

```diff
@@ -13,6 +14,34 @@ The
 %a{href:"https://hourofcode.com"} Hour of Code
 is a global movement reaching tens of millions of students in 180+ countries. Anyone, anywhere can organize an Hour of Code event or try any of the over 200+ one-hour tutorials, available in over 45 languages. Teachers, to view lesson plans and additional teacher resources please select your desired Hour of Code tutorial from the side menu to the left.
 
+#resources.responsive-body
+  .col-50
+    .tutorial-tile
+      %a{href: CDO.studio_url("/s/applab-intro/reset")}
+        %img.tutorial-tile-img{src: CDO.code_org_url("/images/app-lab/applab-intro.png"), style: "width: 100%; height: 195px"}
+      .tutorial-info{style: "height: 200px;"}
+        %h3.tutorial-info-h Intro to App Lab Tutorial (beta)
+        .smalltext Create your own app in JavaScript using block based programming. Or take your skills to the next level with text-based programming.
+        %a{href: CDO.studio_url("/s/applab-intro/reset")}
+          %button.tutorial-gray{style: "margin-top: 20px;"} Start tutorial
+        %div{style: "float: right; margin-top: 25px;"}
```

Comment:
I realize you just copied what I had in /applab, but if we're going to use the same styles inline, we might as well make it a part of a class.